docs/user.md

```diff
@@ -226,25 +284,34 @@ python scripts/dump_processed.py -t all -c INPUT_COFFEA --json ORIGINAL_JSON_INP
 #   -j JSONS, --jsons JSONS
 #                         Original input json files, splitted by ,. Wildcard option * available as well.
 ```
+<details>
+<summary><strong>Quick exercise — postprocessing example (Summer24)</strong></summary>
+
+Try running the following command:
+```python 
+python scripts/dump_processed.py -c Commissioning_tutorial/hists_ctag_DY_sf_data_Summer24_2024_ctag_DY_sf/hists_ctag_DY_sf_data_Summer24_2024_ctag_DY_sf.coffea -n ctag_DY_sf_2024 -t lumi 
+```
+Make sure you activated [BRIL](https://twiki.cern.ch/twiki/bin/viewauth/CMS/BrilcalcQuickStart#Getting_started_without_requirin) environment properly before running the command. 
```

Comment:
Is it necessary to activate the environment? I think it is activated when calling, so this would be misleading.